core/src/main/kotlin/org/evomaster/core/output/TestCaseWriter.kt

```diff
@@ -399,12 +437,49 @@ class TestCaseWriter {
                 lines.append("$path?\" + ")
 
                 lines.indented {
-                    (0 until elements.lastIndex).forEach { i -> lines.add("\"${GeneUtils.applyEscapes(elements[i], mode = GeneUtils.EscapeMode.SQL, format = format)}&\" + ") }
-                    lines.add("\"${GeneUtils.applyEscapes(elements.last(), mode = GeneUtils.EscapeMode.SQL, format = format)}\"")
+                    (0 until elements.lastIndex).forEach { i ->
+                        lines.add(
+                            "\"${
+                                GeneUtils.applyEscapes(
+                                    elements[i],
+                                    mode = GeneUtils.EscapeMode.SQL,
+                                    format = format
+                                )
+                            }&\" + "
+                        )
+                    }
+                    lines.add(
+                        "\"${
+                            GeneUtils.applyEscapes(
+                                elements.last(),
+                                mode = GeneUtils.EscapeMode.SQL,
+                                format = format
+                            )
+                        }\""
+                    )
                 }
             }
         }
-        lines.append(")")
+
+        if (format.isCsharp()) {
+            if (hasBody) {
+                when (verb) {
+                    "post", "put" -> lines.append(", httpContent);")
```

Comment:
what about patch?   better to refactor in a private method called `isVerbWithPossibleBodyPayload(verb: String)`

Reply:
Done